chess/svg.py

```diff
@@ -58,6 +59,13 @@
 }}
 """.format(**DEFAULT_COLORS)
 
+class Arrow():
+    """
+    Details of an arrow to be drawn.
+    """
+    def __init__(self, head, tail):
```

Comment:
This is bikeshedding, but the order `tail -> head` like `Move(from_square, to_square)` seems more natural to me.